Better logging for failed CRL fetch

The CRL fetching seems to be throwing a lot of false-positives from a support perspective.  We should quiet down the warning that appears so that it does not look so intimidating to the end-user.

Currently it looks like this:

```
2017-10-03 18:13:59,273 [ERROR] Error loading CRL from https://crl.qz.io
javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
            at sun.security.ssl.Alerts.getSSLException(Unknown Source)
            at sun.security.ssl.SSLSocketImpl.fatal(Unknown Source)
            at sun.security.ssl.Handshaker.fatalSE(Unknown Source)
            at sun.security.ssl.Handshaker.fatalSE(Unknown Source)
            at sun.security.ssl.ClientHandshaker.serverCertificate(Unknown Source)
            at sun.security.ssl.ClientHandshaker.processMessage(Unknown Source)
            at sun.security.ssl.Handshaker.processLoop(Unknown Source)
            at sun.security.ssl.Handshaker.process_record(Unknown Source)
            at sun.security.ssl.SSLSocketImpl.readRecord(Unknown Source)
            at sun.security.ssl.SSLSocketImpl.performInitialHandshake(Unknown Source)
            at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source)
            at sun.security.ssl.SSLSocketImpl.startHandshake(Unknown Source)
            at sun.net.www.protocol.https.HttpsClient.afterConnect(Unknown Source)
            at sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(Unknown Source)
            at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(Unknown Source)
            at sun.net.www.protocol.http.HttpURLConnection.getInputStream(Unknown Source)
            at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(Unknown Source)
            at java.net.URL.openStream(Unknown Source)
            at qz.auth.CRL$1.run(Unknown Source)
Caused by: sun.security.validator.ValidatorException: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
            at sun.security.validator.PKIXValidator.doBuild(Unknown Source)
            at sun.security.validator.PKIXValidator.engineValidate(Unknown Source)
            at sun.security.validator.Validator.validate(Unknown Source)
            at sun.security.ssl.X509TrustManagerImpl.validate(Unknown Source)
            at sun.security.ssl.X509TrustManagerImpl.checkTrusted(Unknown Source)
            at sun.security.ssl.X509TrustManagerImpl.checkServerTrusted(Unknown Source)
            ... 15 more
Caused by: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
            at sun.security.provider.certpath.SunCertPathBuilder.build(Unknown Source)
            at sun.security.provider.certpath.SunCertPathBuilder.engineBuild(Unknown Source)
            at java.security.cert.CertPathBuilder.build(Unknown Source)
            ... 21 more
```

Although it should probably look something more like this:

```
2017-10-03 18:13:59,273 [WARN] Unable to access CRL from https://crl.qz.io.  PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
```

--- a/src/qz/auth/CRL.java
+++ b/src/qz/auth/CRL.java
@@ -51,7 +51,7 @@ public class CRL {
                         log.info("Successfully loaded {} CRL entries from {}", instance.revokedHashes.size(), CRL_URL);
                     }
                     catch(IOException e) {
-                        log.error("Error loading CRL from {}", CRL_URL, e);
+                        log.error("Error loading CRL from {}, {}", CRL_URL, e.toString());
                     }
                 }
             }.start();

--- a/src/qz/auth/CRL.java
+++ b/src/qz/auth/CRL.java
@@ -18,7 +18,7 @@ public class CRL {
     private static final Logger log = LoggerFactory.getLogger(CRL.class);
 
     /** The URL to the QZ CRL. Should not be changed except for dev tests */
-    public static final String CRL_URL = "https://crl..io";
+    public static final String CRL_URL = "https://crl.qz.io";
 
     private static CRL instance = null;
 

--- a/src/qz/auth/CRL.java
+++ b/src/qz/auth/CRL.java
@@ -51,7 +51,7 @@ public class CRL {
                         log.info("Successfully loaded {} CRL entries from {}", instance.revokedHashes.size(), CRL_URL);
                     }
                     catch(IOException e) {
-                        log.error("Error loading CRL from {}, {}", CRL_URL, e.toString());
+                        log.warn("Error loading CRL from {}, {}", CRL_URL, e.toString());
                     }
                 }
             }.start();

--- a/src/qz/auth/CRL.java
+++ b/src/qz/auth/CRL.java
@@ -18,7 +18,7 @@ public class CRL {
     private static final Logger log = LoggerFactory.getLogger(CRL.class);
 
     /** The URL to the QZ CRL. Should not be changed except for dev tests */
-    public static final String CRL_URL = "https://crl.qz.io";
+    public static final String CRL_URL = "https://crl..io";
 
     private static CRL instance = null;
 
@@ -51,7 +51,7 @@ public class CRL {
                         log.info("Successfully loaded {} CRL entries from {}", instance.revokedHashes.size(), CRL_URL);
                     }
                     catch(IOException e) {
-                        log.warn("Error loading CRL from {}, {}", CRL_URL, e.toString());
+                        log.warn("Unable to access CRL from {}, {}", CRL_URL, e.toString());
                     }
                 }
             }.start();

